index.js

```diff
@@ -46,6 +46,11 @@ module.exports = () => {
 			return topicApi.publish(sender);
 		};
 
+		const cancelScheduledMessages = async (publicationId, sequenceNumber) => {
+			debug(`Cancelling messages ${sequenceNumber} for topic ${publicationId}`);
+			return sendersByPublication[publicationId].cancelScheduledMessages(sequenceNumber);
```

Comment:
Should we check if `sendersByPublication[publicationId]` exists? We could potentially send a wrong `publicationId` so we would be calling `undefined.cancelScheduledMessages(sequenceNumber);`

Reply:
https://github.com/guidesmiths/systemic-azure-bus/pull/72/commits/8ccf8f1f091d8cea48e301e23bd59af935c4f8d5